### DISCUSSION TITLE
dockerized meshcentral behind dockerized nginx

### DISCUSSION THREAD

**Discussion by abdeldjalil-fellah** [Ylianst/MeshCentral#6563] [Q&A]
please provide docker-compose.yml, nginx.conf, config.json to run dockerized meshcentral behind dockerized nginx as mc.example.com

**Comment by si458** [comment]
https://github.com/Ylianst/MeshCentral/tree/master/docker#docker-composeyml
https://ylianst.github.io/MeshCentral/meshcentral/#nginx-reverse-proxy-setup

**Reply by abdeldjalil-fellah** [reply]
interesting! but the docs does not show an example of a dockerized meshcentral

**Reply by si458** [reply]
We know the docs need updating its on the to-do list
